REFERENCE.md

```diff
@@ -414,6 +415,12 @@ Data type: `String[1]`
 
 name of the apache2 service
 
+##### <a name="graph_facts"></a>`graph_facts`
+
+Data type: `Optional[Variant[String[1], Array[String[1]]]]`
+
+A list of the puppet facts to graph.  See ''https://github.com/voxpupuli/puppetboard''
```

Comment:
This file needs to be regenerated since it doesn't match the documentation comment.

Reply:
Pardon my ignorance but I don't know what you mean by "needs to be regenerated."  How does one regenerate (what?) file?

Reply:
`bundle exec rake reference`